libs/SalesforceSDK/src/com/salesforce/androidsdk/app/UUIDManager.java

```diff
@@ -26,55 +26,72 @@
  */
 package com.salesforce.androidsdk.app;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.UUID;
-
-import com.salesforce.androidsdk.analytics.security.Encryptor;
-
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.content.SharedPreferences.Editor;
 
+import com.salesforce.androidsdk.analytics.security.Encryptor;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
 /**
- * Helper class for UUID generation
+ * Helper class for UUID generation.
+ *
+ * @deprecated Do not use this starting with Mobile SDK 6.0. This will be removed
+ * in Mobile SDK 7.0. Use {@link com.salesforce.androidsdk.util.SalesforceKeyGenerator instead}.
  */
 public class UUIDManager {
```

Comment:
I deprecated this class in favor of `SalesforceKeyGenerator`. We need to keep this around for now for upgrade steps.